types/src/contract_event.rs

```diff
@@ -62,8 +52,6 @@ impl Deref for ContractEvent {
 pub struct ContractEventV0 {
     /// The unique key that the event was emitted to
     key: EventKey,
-    /// The number of messages that have been emitted to the path previously
-    sequence_number: u64,
```

Comment:
This is the other big change. Move no longer needs to keep track of this; the storage layer will be responsible for assigning it.